### DISCUSSION TITLE
OBAL Raspi Zero Hat (aka PXF Mini)

### DISCUSSION THREAD

**Discussion by Iniigo** [HefnySco/OBAL#5] [Ideas]
Hello,
I am teaching unmanned related technology in Spain from kids to elders.

- I would like to teach with a common inexpensive Ardupilot FC (due to Ardupilot flash limit FCs become obsolete with time)
that grows from Sub250 grams quad to larger quad ones.
- Also used with surface, wing and sub RC and unmanned platforms.

Due to Erle's business crash PXF Mini seems no longer available.
![Z](https://user-images.githubusercontent.com/96335604/147059815-175ad2b9-b5ff-4b64-bc56-b6bb7422b559.jpg)

I have been messing around many information and that shown in the Ardupilot 2021 Dev Conference Youtube Videos and I believe there is a great opportunity for OBAL to become a Raspi Zero Hat Ardupilot too.
Just to share preliminary thoughts with this Community.
What do you think about that?

Thanks Hefny for so great Project!!
Iñigo

BTW Why not an "OBAL BEE"? [http://luminousbe.es/ardubee/](url)

![9k=](https://user-images.githubusercontent.com/96335604/147060101-9885471d-6a06-4210-a1dd-aab0d7b7b511.jpg)

![2Q==](https://user-images.githubusercontent.com/96335604/147060208-49c33b19-2426-4779-b0d1-ce7370c8bff6.jpg)

**Comment by HefnySco** [comment]
Thank you for your suggestion. Great Idea.
I thought of a fast win which is to reshape OBAL into drone frame. no embedded components ... just the frame.
